src/main/java/seedu/address/logic/commands/edit/EditCommand.java

```diff
@@ -101,14 +101,15 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         assert personToEdit != null;
 
         Name updatedName = editPersonDescriptor.getName().orElse(personToEdit.getName());
-        Phone updatedPhone = editPersonDescriptor.getPhone().orElse(personToEdit.getPhone());
-        Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
+        Phone updatedPhone = editPersonDescriptor.getPhone().or(() ->personToEdit.getPhone()).orElse(null);
+        Email updatedEmail = editPersonDescriptor.getEmail().or(personToEdit :: getEmail).orElse(null);
```

Comment:
syntax should be similar between `updatedPhone`, `updatedEmail`, `updateAddress`